shared/templates/service_disabled/ansible.template

```diff
@@ -11,9 +11,10 @@
         - name: Disable service {{{ SERVICENAME }}}
           systemd:
             name: "{{{ DAEMONNAME }}}.service"
-            enabled: "no"
+            enabled: false
```

Comment:
Thanks. For information, we have an Ansible section in the Project Style Guide:
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible